### PR TITLE
use IdeaCollection instead of Buergerhaushalt workbench / process

### PR DESCRIPTION
**Pull request by der-john**
this continues / contains #2307. It is still simply a reorganisation of the functionality of the Buergerhaushalt. The Kiezkasse functionality will be "merged" into the IdeaCollection soon.

### CODE REVIEW

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Buergerhaushalt/Context/Module.ts [hunk 1]
```diff
@@ -1,32 +1,16 @@
...
         ])
-        .config(["adhEmbedProvider", (adhEmbedProvider : AdhEmbed.Provider) => {
+        .config(["adhEmbedProvider", (adhEmbedProvider: AdhEmbed.Provider) => {
             adhEmbedProvider.registerContext("buergerhaushalt", ["burgerhaushalt"]);
-        }])
-        .config(["adhResourceAreaProvider", (adhResourceAreaProvider : AdhResourceArea.Provider) => {
-            AdhMeinberlinBuergerhaushaltWorkbench.registerRoutes(
-                RIBuergerhaushaltProcess.content_type,
-                "buergerhaushalt"
-            )(adhResourceAreaProvider);
```
Comment (xi) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/Buergerhaushalt/Context/Module.ts]: I think this is still needed, except with AdhMeinberlinIdeaCollection istead of AdhMeinberlinBuergerhaushaltWorkbench.

File: src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Module.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            AdhPermissionsModule.moduleName,
+            AdhProcessModule.moduleName,
+            AdhResourceAreaModule.moduleName,
+            AdhTopLevelStateModule.moduleName
+        ])
+        .config(["adhResourceAreaProvider", "adhConfig", (adhResourceAreaProvider, adhConfig) => {
+            var processType = RIBuergerhaushaltProcess.content_type;
+            var customHeader = adhConfig.pkg_path + IdeaCollection.pkgLocation + "/CustomHeader.html";
+            adhResourceAreaProvider.customHeader(processType, customHeader);
+            IdeaCollection.registerRoutes(processType)(adhResourceAreaProvider);
```
Comment (der-john) [inline src/meinberlin/meinberlin/static/js/Packages/Meinberlin/IdeaCollection/Module.ts]: Doesn't this do the same thing?

**Comment by xi** [comment]
I did not check in detail, but it I think it's fine to merge because we are still working on this.

**Comment by xi** [comment]
jsunit tests are also broken in master. I will investigate.

**Pull request merged**